src/main/java/st/ratpack/auth/CachingTokenValidator.java

```diff
@@ -21,39 +21,23 @@ public CachingTokenValidator(TokenValidator upstreamValidator) {
 		cache = Caffeine.<String, Promise<Optional<OAuthToken>>>newBuilder()
 			.maximumSize(10000L)
 			.expireAfterWrite(5L, TimeUnit.MINUTES)
+			.recordStats()
 			//			.executor(Execution.current().getEventLoop())  Don't do this it makes Ratpack hang.
 			.build();
```

Comment:
I had initially tried this approach but found that `cache.get(token)` wasn't triggering the loader, I'll check again.